src/ssv-network.ts

```diff
@@ -853,6 +855,47 @@ export function handleOperatorRemoved(event: OperatorRemovedEvent): void {
   }
 }
 
+export function handleOperatorWhitelistUpdated(
+  event: OperatorWhitelistUpdatedEvent
+): void {
+  let entity = new OperatorWhitelistUpdated(
+    event.transaction.hash.concatI32(event.logIndex.toI32())
+  )
+  entity.operatorId = event.params.operatorId
+  entity.whitelisted = event.params.whitelisted
+
+  entity.blockNumber = event.block.number
+  entity.blockTimestamp = event.block.timestamp
+  entity.transactionHash = event.transaction.hash
+
+  entity.save()
+
+  let whitelisted = Account.load(event.params.whitelisted)
```

Comment:
Advice: if I am not wrong, this field can be empty, as it was only one event triggered if you set the whitelist to something, or even if you reset it to go back to being Public.
If I am correct, I'd suggest adding a check here to avoid trying to load an empty address, or trying to create an account with an empty address.

And also, this is where you could reset `isPrivate` to `false`.

Reply:
I am still unsure you have addressed this issue. If `events.params.whitelisted` is not set, what happens?

Reply:
yes I didn't, testing it now in a new deployment